mechanics/RELEASE-VERSIONING-PRINCIPLES.md

```diff
@@ -609,276 +546,42 @@ tuple
    </td>
   </tr>
   <tr>
-   <td>v1beta1
-   </td>
-   <td>v1beta1
-   </td>
-   <td>default (deprecated)
-   </td>
-  </tr>
   <tr>
-   <td>v1alpha1
-   </td>
-   <td>Lemonade
-   </td>
-   <td>default 
-<p>
-(deprecated)
-   </td>
-  </tr>
-  <tr>
-   <td rowspan="3" >0.9.x
+   <td rowspan="2" >0.20.x
```

Comment:
Just adding it here, but for Eventing, v1beta1 can be left as optional (deprecated) and maybe in .20 we'll not serve it, but currently it's still being served.